File: 4th/task04.py
class MyClass:
    def __init__ (self, key, strings) :
        self.key = key
        self.strings = strings

    def printStatus (self) :
        print(self.key)
        print(self.strings)

myClass = MyClass (101, "name")

print(myClass.key)
print(myClass.strings)

myClass.printStatus()

File: 4th/task04.js

class MyClass {
	constructor (key, str) {
		this.key = key;
		this.str = str;
	}

	printStatus () {
		console.log(this.key);
		console.log(this.str);
	}
}

myObject = new MyClass(101, "I am a pen.");
myObject.printStatus();

